tests/fixtures/31-mustache_lint-js_token_test.patch

```diff
@@ -0,0 +1,34 @@
+From 7c4ad58c7f7a462e39577959a6c4e05a868cc610 Mon Sep 17 00:00:00 2001
+From: Andrew Nicols <andrew@nicols.co.uk>
+Date: Mon, 8 Aug 2022 15:07:36 +0800
+Subject: [PATCH 1/1] MDL-12345 mustsache: fixture for js parsing failure
```

Comment:
Small typo? `mustsache` => `mustache`

HTH,
Matteo

Reply:
You are right, @scara . Though I must confess how "mustsache" sounded in my brain, LOL.

Reply:
I've taken the liberty of amending the typo adding extra commit, will squash on merge.